src/NzbDrone.Core/Download/Clients/Tribler/Models/TriblerDownloadClientApi.cs

```diff
@@ -0,0 +1,226 @@
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Runtime.Serialization;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace NzbDrone.Core.Download.Clients.Tribler
+{
+    public enum DownloadStatus
+    {
+        [EnumMember(Value = @"WAITING4HASHCHECK")]
+        Waiting4HashCheck = 0,
+
+        [EnumMember(Value = @"HASHCHECKING")]
+        Hashchecking = 1,
+
+        [EnumMember(Value = @"METADATA")]
+        Metadata = 2,
+
+        [EnumMember(Value = @"DOWNLOADING")]
+        Downloading = 3,
+
+        [EnumMember(Value = @"SEEDING")]
+        Seeding = 4,
+
+        [EnumMember(Value = @"STOPPED")]
+        Stopped = 5,
+
+        [EnumMember(Value = @"ALLOCATING_DISKSPACE")]
+        AllocatingDiskspace = 6,
+
+        [EnumMember(Value = @"EXIT_NODES")]
+        Exitnodes = 7,
+
+        [EnumMember(Value = @"CIRCUITS")]
+        Circuits = 8,
+
+        [EnumMember(Value = @"STOPPED_ON_ERROR")]
+        StoppedOnError = 9,
+    }
+
+    public class Trackers
+    {
+        [JsonProperty("url", NullValueHandling = NullValueHandling.Ignore)]
```

Comment:
`url` is automatically handled as `Url`, do we need to specify `NullValueHandling = NullValueHandling.Ignore` for every property here because that hasn't been necessary elsewhere? We also set that as a default when serializing.

I would like to have `JsonProperty` removed from all properties where it's not necessary (just different in case), not going to comment on all the properties, but this applies to them all.

Reply:
should be cleaned up now.

Reply:
Thank you for the feedback btw.